nrf_rpc: made private bt serialization and proxy cb API the public NRF RPC API (#16423)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

nRF RPC libraries
-----------------

> * Converted internal Bluetooth serialization API and Bluetooth callback proxy API into the public NRF RPC API.

Reviewer: ```suggestion
* Updated the internal Bluetooth serialization API and Bluetooth callback proxy API to become part of the public NRF RPC API.
```
preferable wording for the release notes
